src/components/crowdsale/index.js

```diff
@@ -247,7 +246,7 @@ export class Crowdsale extends React.Component {
                   </div>
                   {contributorsBlock}
                 </div>
-                <p className="hash">{`${crowdsaleExecID || (MintedCappedProxy && MintedCappedProxy.addr)}`}</p>
+                <p className="hash">{`${crowdsaleExecID || (contractStore[proxyName] && contractStore[proxyName].addr)}`}</p>
```

Comment:
You can extract `contractStore[proxyName]` to a variable, say: `const proxy = contractStore[proxyName]`?

With that you can pass the `prettier error`.